orchagent/neighorch.cpp

```diff
@@ -1012,9 +1018,9 @@ bool NeighOrch::removeNeighbor(const NeighborEntry &neighborEntry, bool disable)
         if (status != SAI_STATUS_SUCCESS)
         {
             /* When next hop is not found, we continue to remove neighbor entry. */
-            if (status == SAI_STATUS_ITEM_NOT_FOUND)
+            if (status == SAI_STATUS_ITEM_NOT_FOUND || status == SAI_STATUS_INVALID_PARAMETER)
```

Comment:
This seems to be changing existing flow. Are we getting SAI_STATUS_INVALID_PARAMETER here for non-existent entries?

Reply:
Yes https://github.com/sonic-net/sonic-sairedis/blob/60fa682fcce68ef9100bb161a05b19cb5bed0c82/meta/Meta.cpp#L3261